source/velocity_boundary_conditions/interface.cc

```diff
@@ -52,6 +52,39 @@ namespace aspect
 
 
     template <int dim>
+    Tensor<1,dim>
+    Interface<dim>::boundary_velocity (const Point<dim> &) const
+    {
+      /**
+       * We can only get here if the new-style boundary_velocity function (with
+       * two arguments) calls it. This means that the derived class did not override
+       * the new-style boundary_velocity function, and because we are here, it also
+       * did not override this old-style boundary_velocity function (with one argument).
+       */
+      Assert (false, ExcMessage ("A derived class needs to override either the "
+                                 "boundary_velocity() or boundary_velocity(types::boundary_id) function."));
```

Comment:
Please clarify this as `boundary_velocity(position)` or `boundary_velocity(types::boundary_id,position)`.